add method to set fa identifier properly for hbx admins

ZASMan opened the pull request:
ME-180424645

ZASMan committed:
Adds method to set fa identifier properly for hbx admins.

Moves app id lookup.

bill-transue reviewed (changes requested):
Let's check all of a user's family ids when trying to access an application incase they change primary_families.

ZASMan committed:
Add array identifier for financial assistance identifiers.

ZASMan committed:
Fix financial assistance application controller errors.

Pass primary family id through families navigation.

Fixes personal.html.erb spec.

Fix applications setting for index page.

stuff

ZASMan commented:
> Let's check all of a user's family ids when trying to access an application incase they change primary_families.

Fixed

ZASMan committed:
Remove refactored out comment.

bill-transue committed:
Merge branch 'trunk' into 180424645-financial-assistance-identifier-fix

markgoho committed:
Merge branch 'trunk' into 180424645-financial-assistance-identifier-fix

bill-transue reviewed (approved):
Specs and fixes problems finding FA applications because of primary family id changing.
